packages/explorer/src/app/(explorer)/[chainName]/worlds/page.tsx

```diff
@@ -1,15 +1,39 @@
-import { notFound, redirect } from "next/navigation";
+import { Address } from "viem";
+import { supportedChains, validateChainName } from "../../../../common";
+import { WorldsForm } from "./WorldsForm";
 
-export const dynamic = "force-dynamic";
+type ApiResponse = {
+  items: {
+    address: {
+      hash: Address;
+    };
+  }[];
+};
+
+async function fetchWorlds(chainName: string): Promise<Address[]> {
+  validateChainName(chainName);
+
+  const chain = supportedChains[chainName];
+  const blockExplorerUrl = chain.blockExplorers?.default.url;
+  if (!blockExplorerUrl) {
+    return [];
+  }
+
+  const response = await fetch(`${blockExplorerUrl}/api/v2/mud/worlds`);
```

Comment:
do we have a good fallback when this is run locally and this will fail?

Reply:
if you try to access non-supported chains, the UI fails with an error "Unsupported chain ...".

Reply:
Oh but yeah, the fetch could fail even if the chain is supported. Let's catch that error and do not prevent the UI from being displayed